src/helpers.js

```diff
@@ -103,16 +102,10 @@ export default function(t) {
   }
 
   return {
-    buildChildren: buildChildren,
-
-    flattenExpressions: flattenExpressions,
-
-    toFunctionCall: toFunctionCall,
-
-    toFunctionCallStatement: toFunctionCallStatement,
-
-    toReference: toReference,
-
+    buildChildren,
+    flattenExpressions,
+    toFunctionCall,
+    toReference,
```

Comment:
It might be better to export these helpers directly adding `t` to the arguments for each.

```js
export toReference(t, node, identifier) { /* ... */ }
```

We're going to want `file` available for `file.addImport` and that will only work as an extra parameter. Might as well be consistent.